System_Information.bat

```diff
@@ -0,0 +1,29 @@
+@echo off
+
+:: Create directory "System Information" if doesn't exist
+echo Create folder: System Information
+IF NOT EXIST "System Information" md "System Information"
+
+SET filePath="System Information\System_Information.txt"
+
+:: Run SYSTEMINFO
+echo Running command: systeminfo
+echo ------- Output of command: systeminfo --------- > %filePath%
+systeminfo >> %filePath%
+
+:: Run WMIC
+echo Running command: WMIC
+echo ------- Output of command: WMIC --------- >> %filePath%
+wmic /APPEND:%filePath% product get name,vendor,version >> nul
+
+::Run Dxdiag and append data to System_Information.txt
+echo Running command: dxdiag
+echo ------- Output of command: dxdiag --------- >> %filePath%
+dxdiag /T dxdiag.txt
+type dxdiag.txt >> %filePath%
+del dxdiag.txt
+
+::Run PsInfo and append data to System_Information.txt

```

Comment:
Good!

32-bit and 64-bit software is listed in a slightly different registry location (obvious in hindsight). I've combined the two commands, for a batch file:

```
powershell "Get-ItemProperty HKLM:\Software\Microsoft\Windows\CurrentVersion\Uninstall\* | Select-Object DisplayName, DisplayVersion, Publisher, InstallDate | Format-Table -AutoSize | Out-String -Width 5000 | Out-File software.txt" 
type software.txt >> %filePath%

powershell "Get-ItemProperty HKLM:\Software\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall\* | Select-Object DisplayName, DisplayVersion, Publisher, InstallDate | Format-Table -AutoSize | Out-String -Width 5000 | Out-File software32.txt" 2>nul
type software32.txt >> %filePath%

del software.txt software32.txt
```

The first one gets the native bit software, the second gets the 32-bit software and will fail on a 32-bit OS (silently ignored with the stderr redirect)/

At the end, `Out-String -Width 5000 | Out-File software.txt` stops the columns being chopped off (It uses the width of the console even when redirecting to a file!)